modules/telestion-api/src/main/java/de/wuespace/telestion/api/message/header/InformationUtils.java

```diff
@@ -0,0 +1,36 @@
+package de.wuespace.telestion.api.message.header;
+
+import de.wuespace.telestion.api.utils.AbstractUtils;
+import io.vertx.core.MultiMap;
+
+import java.util.Arrays;
+
+/**
+ * Utility class for conveniently adding {@link Information} to a {@link MultiMap header}.
+ *
+ * @author Cedric Boes (@cb0s), Ludwig Richter (@fussel178)
+ */
+public class InformationUtils extends AbstractUtils {
```

Comment:
What? Don't extend static utility classes. You will make your life much harder if you want to evolve the API. Just imagine you want to insert a new function and suddenly some implementations are broken. Final classes give you the freedom to expand your API without worrying. Don't toss it away in a util class for no good reason.